src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
+    private static final Integer MAX_BALL_VALUE = new Random().nextInt(100);
+
+    public String getRandomBall() {
```

Comment:
```suggestion
    public Ball getRandomBall() {
```
замість строки краще повертати об'єкт класу Ball, так з ним зручніше буде працювати якщо потрібно буде змінити дані або частково вивести об'єкт в консоль